### DISCUSSION TITLE
Errors when displaying in IE11, errors came from Commons file inside .next's chunk folder

### DISCUSSION THREAD

**Discussion by Corouna** [vercel/next.js#17553] [Help] [answered]
I have issue with displaying my team's website in IE11. It went fine on other modern browsers tho.

This is the warning that came on IE11 console

![image](https://user-images.githubusercontent.com/4554944/94895974-0b502b00-04bf-11eb-86a0-ace15d7c1ff3.png)

And the line that triggers it

![image](https://user-images.githubusercontent.com/4554944/94896230-92050800-04bf-11eb-8466-98d69ec88a9f.png)

That arrow was not transpiled to ES5 compatible apparently, and there were 2 other instances in the same file where arrow function was used. 

![image](https://user-images.githubusercontent.com/4554944/94896311-c5e02d80-04bf-11eb-9871-a0894060cec8.png)

I saw some other discussions in here regarding issues with IE11, the answer i found was apparently it might be caused by some modules in node_modules were not transpiled to ES5 properly. Now that i found these 3 arrows in Commons, but i couldn't figure out which modules that led to it. 

My only hint was the nearest functions where the arrow appears in the file. Looks like functions that came from @material-ui/core library, but im not sure if MUI was the culprit for this.

So how can i know that error came from which module? I plan to use `next-transpile-modules` plugin to transpile to modules that were not transpiled to ES5 for IE11, but based on the error displayed, i have zero clue as to which modules that causing it. 

I definitely need help on getting this issue out and make the website works in IE11.

**Comment by jamesmosier** [comment, marked as answer]
You'll need to search that `common` bundle for `ZFOp` and try to find some unique code in order to track down which module is causing this. Or search your `node_modules` for `toString(16).toUpperCase()` to see if you can find the module that is causing it.

**Reply by Corouna** [reply]
Assume that i found where the root cause is in the node_modules, say like in `@material-ui/core` Button and Card. So with `next-transpile-modules` do i need to transpile the whole `@material-ui/core` or just `@material-ui/core/button` and `@material-ui/core/card`?

But really find the culprit codes in node_modules for a big project is going to be super hectic.. @@;;

**Reply by jamesmosier** [reply]
I would start by doing `@material-ui/core` to see if it fixes your issue. If it does then try just transpiling `@material-ui/core/button` instead. You can do a a little trial-and-error to determine what you will need and what you don't 😄

**Reply by Corouna** [reply]
@jamesmosier With `next-transpile-modules` do i need to do the polyfill setup as well? This one i mean - https://gitmemory.com/issue/zeit/next.js/7993/537530895

**Reply by jamesmosier** [reply]
Nom, you shouldn't need any polyfills. Next already polyfills many modules for older browsers: https://nextjs.org/docs/basic-features/supported-browsers-features#polyfills

That guide is old and isn't needed in most cases.

**Comment by eatdesignlove** [comment]
I meet the same issue. 
but in my case, because react-hook-form.

[react-hook-form / Browser Support?](https://react-hook-form.com/faqs/#question7)

**Reply by Corouna** [reply]
Did you use next-transpile-modules to fix the issue?
